### ISSUE TITLE
Streamlined way of adjusting image alts

### ISSUE THREAD

**Issue by bfintal** [gambitph/Stackable#900]
Each block has it's own way of doing things for image `alt` attributes.

We need to streamline this to provide an intuitive way to for people to change the alt text of all images.

We need to:
1. Ensure that all images used in blocks provide an `alt` attribute
2. Ensure that all background images used in blocks have `role="image"` and `aria-label="alt text here"` as per https://www.davidmacd.com/blog/alternate-text-for-css-background-images.html
3. When selecting an image, the alt text from the media library should be assigned as the default alt text

Where to adjust the `alt`:
